autoemulate/core/compare.py

```diff
@@ -186,12 +191,18 @@ def probablistic_emulators() -> list[type[Emulator]]:
         return [emulator for emulator in ALL_EMULATORS if emulator.supports_uq]
 
     @staticmethod
-    def list_emulators() -> pd.DataFrame:
+    def list_emulators(subset: str = "default") -> pd.DataFrame:
```

Comment:
Perhaps it would be worth updating the type hint to show the two supported values?
```suggestion
    def list_emulators(subset: Literal["all", "default"] = "default") -> pd.DataFrame:
```